_extras/Recipe.md

```diff
@@ -0,0 +1,76 @@
+### Back-channel communications
+* **Create a channel** or private conversation thread on The Carpentries Slack workspace (or other preferred tool). 
+* **Use your communication channel** to communicate about preparations. This will allow everyone on your instructional team to become familiar with the tool as well as improve your preparations.
+### Documentation
+* **Create an Etherpad** or other collaborative document, and add all vital information to be included in your workshop introduction for learners who arrive late. This should include:
+    * Workshop schedule (including all breaks)
+    * Connection instructions
+    * What to do in the event of a connection failure
+    * How to ask questions
+    * Other instructions related to workshop mechanics, including recommended window arrangement
+    * Links to The Carpentries Code of Conduct as well as your workshop website
+* Add a link to your Etherpad to your **workshop website**.
+* For a self-organised workshop, be sure to submit your [workshop request form](https://amy.carpentries.org/forms/self-organised/).
+* **Email your learners** with everything they need to know in advance of the workshop. 
+    * Our [template for in-person trainings](https://docs.carpentries.org/topic_folders/workshop_administration/email_templates.html#email-learners-before-workshop) may be useful. Much of this info may already be on your Etherpad, so you can be selective about information that needs duplicating.
+### Preparing for Adjustments
+* **Plan to cut content.** Online workshops often run more slowly than they do in-person; sometimes much more slowly. Decisions to cut back should be intentional and collaborative.
+    * Remember that skipping steps in a workflow can create problems with dependencies later on.
+* **Agree on your priorities.** Discussing key objectives and broader goals with your instructional team in advance will help everyone navigate difficult trade-offs when time becomes an issue during a workshop.
+* **Have a backup** for all roles and documents. Technical failures come in all shapes and sizes, and life can still happen, too!
+* **Review your pre-workshop surveys** using the links provided in your email from our workshop administrators. Knowing your audience will help you adjust appropriately to their needs!
+### Update your Software
+* Especially for Zoom, you will want to be using the most recent version and the same version as everyone else. Remind your learners to update as well!
+## 2. Workshop Mechanics
+* **Assign a "Zoom DJ"** to assume the "host" role in the meeting. A Zoom DJ will manage breakouts, waiting rooms, muting, and troubleshooting as needed. This role is typically played by a Supporting Instructor; if/when they choose to teach, plan to hand off this role to another Instructor.
+* **Customize profile names** to clearly indicate instructors and helpers. For example, a helper named Ada might change their profile to read "Helper: Ada (she/her)"
```

Comment:
Within the African context, we seldom use pronouns. Perhaps it might be a good idea to link why pronouns are used?